lib/manageiq/appliance_console/principal.rb

```diff
@@ -15,7 +15,7 @@ def initialize(options = {})
       options.each { |n, v| public_send("#{n}=", v) }
       @ca_name ||= "ipa"
       @realm = @realm.upcase if @realm
-      @name ||= "#{service}/#{hostname}@#{realm}"
+      @name ||= "#{service}/#{hostname}\\@#{realm}"
```

Comment:
If `@name` is presented to the user, we might want to escape these `@` later, before we run the two `AwesomeSpawn.run` commands below.  But for now, I think this is the surgical way to fix this.